### PR TITLE
Added variables and console output for lesson 2 assignment

### PR DESCRIPTION
**Pull request by danewallace**
Declared 4 variables of two different types and performed operations on them. Output results to console.

### CODE REVIEW

**Comment by danewallace** [comment]
Added content from second challenge.

File: Lesson03/exercise-three.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+let arrayOne = ['cats', 'dogs', 'lions', 'tigers', 'bears'];
+
+for (let i = 0; i < arrayOne.length; i++) {
+	console.log(i + ',' + arrayOne[i]);
```
Comment (MadEste) [inline Lesson03/exercise-three.js]: 👍

File: Lesson02/calculator.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+const y = 11;
+const z = 100;
+const stringTest = '101';
+
+console.log(x + y);
+console.log(x * z);
+console.log(x + ' ' + stringTest);
+console.log('x' + ' ' + stringTest);
+console.log(z % x);
+console.log(z % y);
```
Comment (MadEste) [inline Lesson02/calculator.js]: 👍 
Nice work experimenting with the modulo expressions.

File: Lesson02/countdown.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+for (let i = 3; i >= 0; i--) {
+  console.log(i);
```
Comment (MadEste) [inline Lesson02/countdown.js]: 👍

File: Lesson03/exercise-one.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+console.log(arrayOne);
+
+arrayOne.push('sixth');
+console.log(arrayOne);
+
+arrayOne.splice(2, 1);
+console.log(arrayOne);
+
+const arrayItems = arrayOne.join(', ');
+console.log(arrayItems);
```
Comment (MadEste) [inline Lesson03/exercise-one.js]: 👍

File: Lesson03/exercise-two.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+let sentence = 'This is a single sentence';
+let sentenceArray = sentence.split(' ');
+console.log(sentenceArray[2]);
```
Comment (MadEste) [inline Lesson03/exercise-two.js]: 👍

File: Lesson02/canDrive.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+const firstName = 'Dane';
+const lastName = 'Wallace';
+const personsAge = 32;
+const drivingAge = 16;
+let drivingSince = personsAge - drivingAge;
+
+if (personsAge === drivingAge) {
+  console.log(`${firstName} ${lastName} has been old enough to drive for less than one year.`);
+} else if (personsAge > drivingAge) {
+  console.log(`${firstName} ${lastName} has been old enough to drive for ${drivingSince} years.`);
```
Comment (MadEste) [inline Lesson02/canDrive.js]: perfect! nice use of template literals for string concatenation!

File: Lesson04/Assignment01/demo.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+function multiplyNumbers() {
+  let firstInput = prompt('Enter a number');
+  let secondInput = prompt('Enter another number');
```
Comment (MadEste) [inline Lesson04/Assignment01/demo.js]: This is for the bonus challenge, but the idea is the same as the assignment and everything looks good, nice work!

File: Lesson04/Assignment01/demo.js [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+
+  let result = 0;
```
Comment (MadEste) [inline Lesson04/Assignment01/demo.js]: can be combined with line 7

**Comment by danewallace** [comment]
Thanks @MadEste, that got me on the right track.

File: Lesson05/Assignment/main.js [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+      break;
+    }
+    case 'B', 'b': {
+      viewBalance();
+      break;
+    }
+    case 'Q', 'q': {
+      alert('You have exited. Thank you.')
+      break;
+    }
```
Comment (MadEste) [inline Lesson05/Assignment/main.js]: I would consider adding one more default switch to catch typos etc and call optionsPrompt() again.

File: Lesson05/Assignment/main.js [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+  } else {
+    balance -= Number(withdrawAmount);
+    alert('You have withdrawn $' + withdrawAmount);
+    optionsPrompt();
+  }
+}
+
+function depositMoney() {
+  let depositAmount = prompt('Please enter amount to deposit:');
+  if (Number(depositAmount) > 50000) {
```
Comment (MadEste) [inline Lesson05/Assignment/main.js]: Nice check!

File: Lesson05/Assignment/main.js [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+    'W to withdraw' + '\n' +
+    'D to deposit' + '\n' +
+    'B to view balance' + '\n' +
+    'Q to quit'
+  );
+  selectOption(input);
+}
+
+function withdrawMoney() {
+  let withdrawAmount = prompt('Please enter amount to withdraw:');
```
Comment (MadEste) [inline Lesson05/Assignment/main.js]: Looks Good!

**Pull request closed without merging**